### PR TITLE
flask command

### PR DESCRIPTION
**Pull request by i-oden**

### CODE REVIEW

File: dds_web/__init__.py [hunk 1]
```diff
@@ -383,22 +383,28 @@ def update_uploaded_file_with_log(project, path_to_log_file):
     """Update file details that weren't properly uploaded to db from cli log"""
     import botocore
     from dds_web.database import models
-    from dds_web.api.project_schemas import verify_project_exists
+    from dds_web.api.schemas.project_schemas import verify_project_exists
```
Comment (aanil) [inline dds_web/__init__.py]: I guess we don't need this if its not used?

File: dds_web/__init__.py [hunk 2]
```diff
@@ -410,31 +416,23 @@ def update_uploaded_file_with_log(project, path_to_log_file):
...
+                        public_key=vals["public_key"],
+                        salt=vals["salt"],
+                        checksum=vals["checksum"],
                     )
-                files_added.append(file)
+                    db.session.add(new_file)
+                    files_added.append(new_file)
+
     db.session.commit()
     flask.current_app.logger.info(f"Files added: {files_added}")
```
Comment (aanil) [inline dds_web/__init__.py]: I'm not sure how the models.File object would look when printed out.

File: dds_web/__init__.py (hunk 1 above)
Comment (i-oden) [inline dds_web/__init__.py]:
```suggestion
```

File: dds_web/__init__.py (hunk 2 above)
Comment (i-oden) [reply dds_web/__init__.py]: There should be a str function in the models file. It'll look like an object

File: dds_web/__init__.py [hunk 3]
```diff
@@ -383,22 +383,27 @@ def update_uploaded_file_with_log(project, path_to_log_file):
     """Update file details that weren't properly uploaded to db from cli log"""
     import botocore
     from dds_web.database import models
-    from dds_web.api.project_schemas import verify_project_exists
     from dds_web import db
     from dds_web.api.api_s3_connector import ApiS3Connector
     from dds_web.api.schemas import file_schemas
```
Comment (aanil) [inline dds_web/__init__.py]:
```suggestion
```

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**